target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -232,6 +232,9 @@ ramips_board_detect() {
 	*"JHR-N926R")
 		name="jhr-n926r"
 		;;
+	*"ZyXEL Keenetic Viva")
+		name="kng_rc"
```

Comment:
Address this to NDM. Reference device table here: https://github.com/ndmsystems/packages/wiki/NDM-Systems-Buildroot#configuration